src/protagonist.h

```diff
@@ -48,7 +48,7 @@ namespace protagonist {
 
         // Wraps snowcrash::Warnings and snowcrash:Blueprint into report object
         // Note: snowcrash::Result::Error is being sent separately as Error object
-        static v8::Local<v8::Object> WrapResult(const snowcrash::ParseResult<snowcrash::Blueprint>& parseResult,
+        static v8::Local<v8::Object> WrapResult(snowcrash::ParseResult<snowcrash::Blueprint>& parseResult,
```

Comment:
Well, if the `parseResult` is mutated (output) then I would put it last. Unless there is some other way. I leave this up to you and @klokane whether you want to honor the immutable parameters first and mutable last ...